### ISSUE TITLE
Having issues with the export in jpeg format

### ISSUE THREAD

**Issue by Majpuc** [ilastik/webilastik#5]
After segmentation is completed, export in raw format is working but not the jpeg format. Test was performed on March 1st  and 2nd on JUSUF. Session ID was e.g.  f86ee079-afca-4662-91f2-c44a2e53d90d (if this helps?)

![image](https://user-images.githubusercontent.com/46992863/222442755-bf93a31a-4ef4-4059-bc9c-55e0099f72ab.png)

**Comment by Tomaz-Vieira** [comment]
Thanks for the bug report =)
I'll look into it, but now that I think about it, maybe we shouldn't even have the jpeg option enabled afterall. It would probably produce some compression artifacts that would be misinterpreted by Nutil. Do you have a particular use case in mind that might benefit from the jpeg compression?

**Comment by Majpuc** [comment]
Not really. We did some export tests today as well and got the NG files we need for the converter using "single segmentations" and "raw" export. Thanks!
